src/applications/edu-benefits/tests/1995/02-stem-all-fields.e2e.spec.js

```diff
@@ -24,90 +24,75 @@ module.exports = E2eHelpers.createE2eTest(client => {
   E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
 
   // Veteran information page.
+  E2eHelpers.expectLocation(client, '/applicant/information');
   client.waitForElementVisible(
     'input[name="root_veteranFullName_first"]',
     Timeouts.slow,
   );
   EduHelpers.completeApplicantInformation(client, testData.data, 'veteran');
   client.axeCheck('.main').click('.form-progress-buttons .usa-button-primary');
-  E2eHelpers.expectNavigateAwayFrom(client, '/applicant/information');
 
   // Benefits eligibility page.
+  E2eHelpers.expectLocation(client, '/benefits/eligibility');
   client.waitForElementVisible('label[for="root_benefit_0"]', Timeouts.slow);
   EduHelpers.completeBenefitsSelection(client, testData.data);
   client.axeCheck('.main').click('.form-progress-buttons .usa-button-primary');
-  E2eHelpers.expectNavigateAwayFrom(client, '/benefits/eligibility');
 
   // STEM page
-  client.waitForElementVisible(
-    'label[for="root_isEdithNourseRogersScholarshipYes"',
-    Timeouts.slow,
-  );
+  E2eHelpers.expectLocation(client, '/benefits/stem');
   Edu1995Helpers.completeStemSelectionFor1995s(client);
-  client.waitForElementVisible(
-    'label[for="root_isEnrolledStemYes"]',
-    Timeouts.slow,
-  );
   Edu1995Helpers.completeStemEnrollmentSelection(client);
-  client.waitForElementVisible(
-    'label[for="root_isPursuingTeachingCertYes"]',
-    Timeouts.slow,
-  );
-  Edu1995Helpers.completePursuingTeachingCertSelection(client);
+  Edu1995Helpers.completeExhaustionOfBenefits(client);
   client.axeCheck('.main').click('.form-progress-buttons .usa-button-primary');
-  E2eHelpers.expectNavigateAwayFrom(client, '/benefits/stem');
 
   // Active duty page
+  E2eHelpers.expectLocation(client, '/active-duty');
```

Comment:
It looks like this is failing an e2e test.

Reply:
Fixed